setup_job.sh

```diff
@@ -10,34 +10,41 @@ set -e
 TOKEN='FILL_ME_IN'
 # bespin-api url
 BASE_URL=http://127.0.0.1:8000/api
+FORMAT_PARAM=format=json
```

Comment:
As of cd6bba647a2041d6e6ff99158d31469b4259dde9, the default response should again be JSON. And while requesting the format as a query parameter, it's best to do this with an "Accept: application/json" header.